src/views/SelectProduct.vue

```diff
@@ -291,14 +291,14 @@ export default defineComponent({
       return {
         included: {
           tags: {
-            list: includedTagsAndOperator.tags,
-            operator: includedTagsAndOperator.operator
+            list: includedTagsAndOperator.tags ? includedTagsAndOperator.tags : [],
+            operator: includedTagsAndOperator.operator ? includedTagsAndOperator.operator : ''
```

Comment:
I think operator will always be there

Reply:
https://github.com/hotwax/threshold-management/blob/main/src/store/modules/job/getters.ts#L47